06_modules_lab/03.py

```diff
@@ -1,11 +1,17 @@
-""" Write a program that searches current working directory
-for files larger than 1MB. Every time you find such a file print
-its name to the user.
-
-- When the program finds a large file. It should ask the user
-  a message asking if she wants to delete it, and delete the
-  file if requested
-
-- Take threshold and path as command line arguments
-"""
-
+import os
+import sys
+import argparse
+

```

Comment:
The code is short and already getting a bit hard to read. Some thoughts here:

1. Use meaningful names in the arguments (path instead of p, size instead of s)
2. Use blank lines between blocks of code: A blank line between the arguments related code and the actual iteration can make a huge difference here